docs/templates/registry.md

```diff
@@ -241,25 +254,24 @@ $ dm deploy <template-name>:<version>
 ```
 
 To resolve the template reference, `dm` looks for a template version directory
-with the given version in the template directory with the given template name.
+with the given version in the template directory with the given template name in
+the default template registry.
 
-By default, it uses the Kubernetes Template Registry. However, you can set a
-different default using the `--registry` flag:
+The default is the [Kubernetes Template Registry](https://github.com/kubernetes/deployment-manager/tree/master/templates),
```

Comment:
Doesn't this now need to point to application-dm-templates as per your change in the dm default?